statistics/handle/handle.go

```diff
@@ -524,10 +537,17 @@ func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg
 	if err != nil {
 		return
 	}
-	data, err := statistics.EncodeCMSketch(cms)
+	data, topNData, err := statistics.EncodeCMSketch(cms)
 	if err != nil {
 		return
 	}
+	for i, v := range topNData {
+		insertSQL := fmt.Sprintf("replace into mysql.stats_topnstore (table_id, is_index, hist_id, value_id, content) values (%d, %d, %d, %d, X'%X')", tableID, isIndex, hg.ID, i, v)
```

Comment:
We should first delete all the old rows or there may be some rows that are not replaced.